module/applications/actor/group-sheet.mjs

```diff
@@ -276,6 +277,8 @@ export default class GroupActorSheet extends ActorSheetMixin(ActorSheet) {
       inputs.focus(ev => ev.currentTarget.select());
       inputs.addBack().find('[type="text"][data-dtype="Number"]').change(this._onChangeInputDelta.bind(this));
       html.find(".action-button").click(this._onClickActionButton.bind(this));
+    } else {
+      html.find(".rest-button").click(this._onClickActionButton.bind(this));
```

Comment:
These buttons only exists for GMs, right? Seems like this would do nothing then.

Reply:
Technically it would allow using rests for group actors in a compendium, but probably not a useful use case.